great_expectations/dataset/sqlalchemy_dataset.py

```diff
@@ -718,11 +718,12 @@ def _get_dialect_type_module(self):
             pass
 
         # Bigquery works with newer versions, but use a patch if we had to define bigquery_types_tuple
-        if (isinstance(self.engine.dialect, pybigquery.sqlalchemy_bigquery.BigQueryDialect) and
-                bigquery_types_tuple is not None):
-            return bigquery_types_tuple
-
-        return self.dialect
+        if (
+                pybigquery is not None and
+                isinstance(self.engine.dialect, pybigquery.sqlalchemy_bigquery.BigQueryDialect) and
+                bigquery_types_tuple is not None
+        ):
+            return self.dialect
```

Comment:
I think we need to keep the `return self.dialect` as the default type, and keep the `return bigquery_types_tuple` for the case where bigquery is installed and it is a bigquery dialect but we had to define that (which happens with an older driver version). So I suggest instead the below, which mimics the try/catch around redshift:

```suggestion
        try:
            if (isinstance(self.engine.dialect, pybigquery.sqlalchemy_bigquery.BigQueryDialect) and
                    bigquery_types_tuple is not None):
                return bigquery_types_tuple
        except (TypeError, AttributeError):
            pass

        return self.dialect
```